packages/react-components/react-list-preview/src/components/List/useListStyles.styles.ts

```diff
@@ -1,33 +1,50 @@
-import { makeStyles, mergeClasses } from '@griffel/react';
 import type { SlotClassNames } from '@fluentui/react-utilities';
-import type { ListSlots, ListState } from './List.types';
+import { makeStyles, makeResetStyles, mergeClasses, shorthands } from '@griffel/react';
+import { ListSlots, ListState } from './List.types';
 
 export const listClassNames: SlotClassNames<ListSlots> = {
   root: 'fui-List',
-  // TODO: add class names for all slots on ListSlots.
-  // Should be of the form `<slotName>: 'fui-List__<slotName>`
 };
 
+const useRootBaseStyles = makeResetStyles({
+  ...shorthands.padding(0),
+  ...shorthands.margin(0),
```

Comment:
```suggestion
  padding:0,
  margin: 0,
```

No need to expand in `makeResetStyles` call